src/services/superfluidMinionService.js

```diff
@@ -0,0 +1,142 @@
+import Web3 from 'web3';
+
+import SuperfluidMinionAbi from '../contracts/superfluidMinion.json';
+import { SUPERFLUID_MINION_STREAMS } from '../graphQL/superfluid-queries';
+import { TokenService } from './tokenService';
+import { chainByID, getGraphEndpoint } from '../utils/chain';
+import { graphFetchAll } from '../utils/theGraph';
+
+import SuperfluidSDK from '@superfluid-finance/js-sdk';
+
+let sf = null;
```

Comment:
This will need to change. The idea of the services is to keep things stateless. Creating this closure will cause state from one TX to the next. If that is desired, move this to react state.

Reply:
this will be reviewed again in the next PR update